[FIX/ADD] Fixed missing steps to instructions + added extra notation

RaymondDashWu opened the pull request:
Added missing steps + comments

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/uber/hubble.gl?pullRequest=13) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/uber/hubble.gl?pullRequest=13) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/uber/hubble.gl?pullRequest=13) it.</sub>

chrisgervang commented on CONTRIBUTING.md:
```diff
@@ -12,9 +12,31 @@ Building Hubble.gl locally from the source requires node.js `>=10`.
+
+# Start up hubble.gl example project
+# Optional: cd to a directory within the examples folder
+yarn start-local
```
I think once in an example folder they would need to run `yarn` again before `yarn start-local`. Can you confirm?

RaymondDashWu replied:
Confirmed. Examples would not run unless I installed dependencies again per directory. Fixed.

RaymondDashWu commented:
Split previous step 5 into 2 steps so that they could be referred to in step 9 and added clarification on that step 

```
# Optional: cd to a directory within the examples folder. If you do, repeat step 5 before doing this next step
```

chrisgervang commented on hubble-playground/.env:
```diff
@@ -0,0 +1 @@
+REACT_APP_MAPBOX_ACCESS_TOKEN='pk.eyJ1IjoicmF5bW9uZGRhc2h3dSIsImEiOiJja2M2bTAzeDMwMW1tMnNtZjEyYXg4NDJ2In0.wtXUl5ZNiG8bssZMriO_cw'
```
Oh, you'll want to remove this and refresh your access token. I haven't used `.env` files before, is this a node feature? Might want to add `.env` to `.gitignore` to prevent future issues.

chrisgervang commented on hubble-playground/.vercel/README.txt:
```diff
@@ -0,0 +1,11 @@
+- The ID of the user or team your Vercel project is owned by ("orgId")
+
+> Should I commit the ".vercel" folder?
+No, you should not share the ".vercel" folder with anyone.
```
Might need to add `.vercel` to `.gitignore` too
